packages/@aws-cdk/aws-eks-legacy/lib/kubectl-layer.ts

```diff
@@ -19,7 +19,7 @@ export interface KubectlLayerProps {
  *
  * @see https://github.com/aws-samples/aws-lambda-layer-kubectl
  */
-export class KubectlLayer extends Construct implements lambda.ILayerVersion {
+export class KubectlLayer extends Resource implements lambda.ILayerVersion {
```

Comment:
Does this mean it becomes basically impossible to subclass `Construct`, and you should always be subclassing `Resource`? Or what's the rationale here?

Reply:
No. This was just a mistake (`ILayerVersion` extends `IResource`, so `KubectlLayer` should also extend `Resource`). It's just that this change surfaced this error.